testing/model-mesh/4-deploy-many-ns-many-models.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source "$THIS_DIR/config.sh"
+
+
+for i in $(seq 1 ${NS_COUNT})
+do
+    NS=${NS_BASENAME}-${i}
+    oc create ns ${NS}
+
+    oc apply -f ${THIS_DIR}/minio-secret.yaml -n ${NS}
+
+    for j in $(seq 1 ${MODEL_COUNT})
+        do sed s/example-onnx-mnist/example-onnx-mnist-${j}/g ${THIS_DIR}/triton.yaml | oc apply -n ${NS} -f -
+    done
+
+    unset NS
+done
+
+# we do not wait for model mesh pods to appear sequentially.
+# we'd rather create mm instances in all namespaces then check for their status.
+
+for i in $(seq 1 ${NS_COUNT})
+do
+    NS=${NS_BASENAME}-${i}
+
+    until [[ "$(oc get pods -n ${NS} | grep '4/4' |grep Running |wc -l)" == ${MM_POD_COUNT} ]]
+    do
+        echo "NS:${NS}: Waiting for the model mesh pods"
+        sleep 1
+    done
```

Comment:
you'll need a timeout here, so that the script terminates (and collects the artifacts even if something wrong happens)

Reply:
I think this is solved by the latest iteration, thanks!